README.md

```diff
@@ -1319,23 +1321,63 @@ features: {
 
 Events published by the widget. Subscribe to these events using [on](#onevent-callback-context).
 
-- **pageRendered** - triggered when the widget transitions to a new page, and animations have finished.
+### ready
 
-    ```javascript
-    // Overriding the "Back to Sign In" click action on the Forgot Password page
-    signIn.on('pageRendered', function (data) {
-      if (data.page !== 'forgot-password') {
-        return;
-      }
-      var backLink = document.getElementsByClassName('js-back')[0];
-      backLink.addEventListener('click', function (e) {
-        e.preventDefault();
-        e.stopPropagation();
-        // Custom link behavior
-      });
-    });
-    ```
-- **passwordRevealed** - triggered when the show password button is clicked.
+Triggered when the widget is ready to accept user input for the first time. Returns a `context` object containing the following properties:
+
+- **controller** - Current controller name
+
+```javascript
+signIn.on('ready', function (context) {
+  if (context.controller === 'primary-auth') {
```

Comment:
could `ready` event happen at other context rather than `primary-auth`?

Reply:
Yep. If the Widget is passed in a `recovery`, `activation`, or `stateToken`, the context will be mapped to that page. For example:

![Example](http://g.recordit.co/N0YKyWrEFW.gif)

Reply:
then let's add a test other than the ready page is primary auth?

Reply:
Done! Added a few more for coverage.